scout/server/blueprints/public/controllers.py

```diff
@@ -0,0 +1,161 @@
+import logging
+from functools import reduce
```

Comment:
The content of these pages are no public any more, so I think it would be more correct to rename the whole blueprint to something else than "public", "index" perhaps?

Reply:
Yes, that's a good idea. I suggest a separate PR for this.

Reply:
Ok!